report/assets/styles.css

```diff
@@ -370,6 +371,10 @@
   color: var(--color-informative);
 }
 
+.lh-audit-group a {
+  color: var(--color-blue-primary);
```

Comment:
I see the link color is intentional, but we have two different link colors now. Should this color be used report-wide?